### DISCUSSION TITLE
Got approved for the education program, but still no access to benefits

### DISCUSSION THREAD

**Discussion by thepxscxl** [community/community#145885] [GitHub Education]
### Hi everyone,

I got accepted for GitHub education some days ago and I'm still waiting to be able to use my benefits.
Problem is that I really need to extend my JetBrains education program and that's why I need the benefits asap.

Is there any way to speed up the process?

Sincerely
Pascal

**Comment by davevad93** [comment]
Hi @thepxscxl ,

GitHub no longer provide manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @thepxscxl  was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by thepxscxl** [comment]
Yeah, that's why I'm writing here. The FAQ says I should write a discussion here, if it takes more than 3 days xd

**Reply by davevad93** [reply]
@thepxscxl , you just have to wait as manual reviews are no longer available, you'll surely have your benefits by next week, don't worry! 👍
